src/containers/DeleteButtons.js

```diff
@@ -33,9 +33,9 @@ export const DeleteActionsButton = withRouter(
     (dispatch, { remediation, issues, afterDelete }) => ({
       onDelete: async () => {
         await Promise.all(
-          issues.map((issueId) =>
-            dispatch(deleteRemediationIssue(remediation.id, issueId))
-          )
+          issues.map((issueId) => {
```

Comment:
I think you have to return the dispatch value. Otherwise, you are not adding any promise array into the `Promise.all` call.